scripts/fractal.js

```diff
@@ -0,0 +1,19 @@
+const concurrently = require('concurrently');
+
+concurrently(
+  [
+    {
+      name: 'stencil',
+      command: 'yarn workspace @dso-toolkit/core stencil build --watch --serve --no-open',
```

Comment:
werkt dat `--no-open` wel? ik meende toch echt stencil wel te zien langskomen?

Reply:
of ik haal development modi door elkaar

Reply:
Ik zal er nog een keer aandachtig naar kijken